CHANGES.md

```diff
@@ -1,6 +1,8 @@
 # Changes
 
 This document describes the relevant changes between releases of the API model.
+## 0.0.343 December 12 2023
+- Remove the `Fullname` search parameter.
```

Comment:
I'd add "from `status_board` model" or something to be more specific

Reply:
Thank you. :-) Updated.